.github/workflows/ci.yaml

```
@@ -193,8 +196,10 @@
           if [[ "${{ needs.format-check.result }}" != "success" || 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)